samples/client/petstore/java/retrofit2rx2/src/main/java/io/swagger/client/JSON.java

```diff
@@ -39,8 +46,42 @@
     private OffsetDateTimeTypeAdapter offsetDateTimeTypeAdapter = new OffsetDateTimeTypeAdapter();
     private LocalDateTypeAdapter localDateTypeAdapter = new LocalDateTypeAdapter();
 
+    public static GsonBuilder createGson() {
```

Comment:
@wing328 I'm talking about this. You can see use of Gson in other modifier files too.

Reply:
The retrofit generator does use gson. I guess this change is due to regenerating all the Java samples.

Reply:
ok no problem then. :)